freud/util/export-ManagedArray.h

```diff
@@ -7,6 +7,7 @@
 #include "ManagedArray.h"
 #include "VectorMath.h"
 
+#include <complex>
```

Comment:
Try moving this to `freud/util/export-ManagedArray.cc`. If that doesn't work, we will need to ignore the clang-tidy lint on this line.